lib/screens/v2/candidate_generator/dup/departures.ex

```diff
@@ -231,6 +237,21 @@ defmodule Screens.V2.CandidateGenerator.Dup.Departures do
     end
   end
 
+  # Determines if the active alerts for a section apply to all routes that are enabled in the section
+  defp section_routes_disabled?(params, alert_informed_entities) do
+    case alert_informed_entities do
+      [] ->
+        false
+
+      _ ->
+        InformedEntity.all_routes_represented?(
+          params.route_ids,
```

Comment:
This won't necessarily work, since configuring sections with a specific list of routes is not required (and, really, we'd _prefer_ not doing so if it's possible to get the same departures with only a stop ID and perhaps a route type filter, since that's more resilient to changes in service). I think what we want here is to pass in the `routes` field of the "section data" passed as the first argument to `get_section_instance`, which is populated via some data fetching performed in `get_routes_serving_section`.

Reply:
Thanks for the callout! Updated to use the fetched routes instead of the routes from the configuration.